docs/WebsocketAPIV5.md

```diff
@@ -0,0 +1,235 @@
+## WebSocket protocol v5 details
+
+WebSocket Protocol v5 requires an Exasol version of at least 8.32.0. It follows the RFC 6455 document of the IETF.
+
+The Exasol connection server identifies the initial GET request by the client.
+This request contains information about the used protocol version.
+Depending on this information the matching login and protocol class is
+chosen.
+
+After the handshake the process is identical to a connection using the
+standard drivers like JDBC or ODBC: The connection server listens to
```

Comment:
`JDBC or ODBC: The connection`

Colon is intended or Period

Reply:
Colon. ;-)